.github/workflows/build-assets.yml

```diff
@@ -149,11 +157,11 @@ jobs:
         run: |
           sudo rm -f ./build/CMakeCache.txt
           sudo rm -rf ./build/CMakeFiles
-          docker_label=${{ secrets.DOCKER_USERNAME }}/build-${{ matrix.os }}:${{ steps.vars.outputs.branch_name }}
+          docker_label=${{ secrets.DOCKER_USERNAME }}/build-${{ matrix.os }}:${{ steps.vars.outputs.community_tag }}
```

Comment:
question: this is the LN cmake step, should this be internal_tag?

Reply:
No - its using the build docker image from community to build the LN sources.

Reply:
ok